packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -187,11 +187,18 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
       onOverlayClickNoCalendar={() => {
         router.push("/apps/categories/calendar");
       }}
-      onClickOverlayContinue={() => {
-        const newUrl = new URL(`${WEBAPP_URL}/login`);
-        newUrl.searchParams.set("callbackUrl", window.location.pathname);
-        newUrl.searchParams.set("overlayCalendar", "true");
-        router.push(newUrl.toString());
+      onClickOverlayContinue={(provider: "calcom" | "google" = "calcom") => {
+        const currentUrl = new URL(`${WEBAPP_URL}/login`);
+        currentUrl.searchParams.set("overlayCalendar", "true");
+        if (provider === "google") {
+          const url = new URL("/getting-started/connected-calendar", window.location.origin);
+          url.searchParams.set("callbackUrl", currentUrl.toString());
+          signIn("google", { callbackUrl: url.toString() });
+        } else {
+          currentUrl.pathname = "/login/";
+          currentUrl.searchParams.set("callbackUrl", window.location.pathname);
```

Comment:
When clicking this option it brings me to the event type page and not back to the booking page.

Reply:
Fixed.

https://www.loom.com/share/3bb001e7dbfc44d8aae01b4a8991ba04?sid=ae88ac67-8653-4799-a6e9-efdec0a040bc

https://www.loom.com/share/1528030b1036499caa2ad1632cd813eb?sid=fdd37c38-8d9b-498f-8e36-f3c04fbe8dab